packages/kbn-test/src/functional_test_runner/cli.ts

```diff
@@ -27,6 +27,12 @@ const parseInstallDir = (flags: Flags) => {
 };
 
 export function runFtrCli() {
+  const runStartTime = Date.now();
+  const toolingLog = new ToolingLog({
+    level: 'info',
+    writeTo: process.stdout,
+  });
+  const reportTime = getTimeReporter(toolingLog, 'functional_test_runner');
```

Comment:
```suggestion
  const reportTime = getTimeReporter(toolingLog, 'scripts/functional_test_runner');
```

Reply:
Why don't we just move this into the run so we don't need to create an additional toolingLog?

Reply:
I'd rather do this in a separate PR, if you think this important